src/write/op.rs

```diff
@@ -472,6 +487,18 @@ enum Operation {
     ///
     /// Represents `DW_OP_GNU_parameter_ref`.
     ParameterRef(UnitEntryId),
+    /// The index of a local in the currently executing function.
+    ///
+    /// Represents `DW_OP_WASM_location 0x00`.
+    WasmLocal(u32),
+    /// The index of a global.
+    ///
+    /// Represents `DW_OP_WASM_location 0x01`.
+    WasmGlobal(u32),
```

Comment:
Currently always 0x01. I don't know the history on why 0x03 was added. My guess is that it is for cases where you don't know the index yet, so you need to be sure there is enough space to write the index later. If so, that will need to use a different variant that somehow records the location of the index so that you know where to write it later.